packages/theme/nuxt.config.js

```diff
@@ -132,6 +134,12 @@ export default {
       })
     ]
   },
+  router: {
+    extendRoutes(routes) {
+      getRoutes(`${__dirname}/_theme`)
+        .forEach((route) => routes.unshift(route));
+    }
+  },
```

Comment:
Is that neccessary?

Reply:
Yes, I need to have a checkout flow start from customer not shipping and in order to do that I have to extend default routes.